src/include/duckdb/execution/operator/persistent/csv_scanner/base_csv_reader.hpp

```diff
@@ -30,19 +30,19 @@ struct StrpTimeFormat;
 class FileOpener;
 class FileSystem;
 
-enum class ParserMode : uint8_t { PARSING = 0, SNIFFING_DIALECT = 1, SNIFFING_DATATYPES = 2, PARSING_HEADER = 3 };
+enum class ParserMode : uint8_t { PARSING = 0, SNIFFING_DATATYPES = 1, PARSING_HEADER = 2 };
```

Comment:
Small note here, independent but inspired from this PR: it would be useful for stable (de)serialisation across version to have a way to signify: this enum value is to be skipped AND should not be reused any more. Otherwise removing / renumbering enums might results in incompatibilities between versions.